sphinx/ext/autosummary/__init__.py

```diff
@@ -324,7 +324,7 @@ def get_items(self, names):
             # -- Grab the summary
 
             documenter.add_content(None)
-            doc = list(documenter.process_doc([self.result.data]))
+            doc = self.result.data
```

Comment:
I think this fix is correct. Double calling `process_doc` is caused by leaving a call of `process_doc` which became unnecessary on #1444.